### PR TITLE
Support reserved words

### PR DESCRIPTION
**Pull request by ngjaying**

### CODE REVIEW

**Comment by jinfahua** [comment]
@ngjaying How to handle below problem?

image

**Comment by ngjaying** [comment]
> @ngjaying How to handle below problem?
> 
> image

Add backtick for the reserved words. meta(`Light-Dimming`->Device)

File: docs/en_US/sqls/query_language_elements.md [hunk 1]
```diff
@@ -13,7 +13,12 @@ Kuiper provides a variety of elements for building queries. They are summarized
 | [ORDER BY](#ORDER BY) | Order the rows by values of one or more columns.             |
 | [HAVING](#HAVING)     | HAVING specifies a search condition for a group or an aggregate. HAVING can be used only with the SELECT expression.             |
 |                       |                                                              |
+### Use reserved words
+These elements and other predefined operators, functions are reserved words. To use the reserved words as the column name and the table name etc., they need to be quoted by backtick.
```
Comment (jinfahua) [inline docs/en_US/sqls/query_language_elements.md:17]: If the field name includes Chinese, or blank character, is it supported? If we support that, we should write the doc as well.

File: xsql/parser_test.go [hunk 2]
```diff
@@ -26,7 +26,18 @@ func TestParser_ParseStatement(t *testing.T) {
 				Sources: []Source{&Table{Name: "tbl"}},
 			},
 		},
-
+		{
+			s: "SELECT `select` FROM tbl",
```
Comment (jinfahua) [inline xsql/parser_test.go:30]: 1. Add parse error unit case - such as 'SELECT `select FROM tbl'
2. Add Chinese, and field name with blank character etc.

File: xsql/processors/xsql_processor_test.go [hunk 3]
```diff
@@ -123,7 +138,8 @@ func createStreams(t *testing.T) {
 	}
 	demo1 := `CREATE STREAM demo1 (
 					temp FLOAT,
-					hum BIGINT,
+					hum BIGINT,` +
+		"`from`" + ` STRING,
```
Comment (jinfahua) [inline xsql/processors/xsql_processor_test.go:142]: Is it to add a field that is record type, and the record type also uses special key-words or characters. Such as,

- select STRUCT(`字段` STRING, `file name` BIGINT),

**Pull request merged**